start/sw/gui-app/electron.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron
+subsection: gui-app
+section: start-sw
+description: 
+---
+
+# Create an application using Electron
+
+[Electron][0] is cross platform framework for creating desktop applications using
```

Comment:
Does this actually work with Jekyll?
Have you tried generating the web locally?

Reply:
I just checked and it does